src/commands/deploy.rs

```diff
@@ -93,6 +93,10 @@ pub struct DeployCommand {
         env = DEPLOYMENT_ENV_NAME_ENV
     )]
     pub deployment_env_id: Option<String>,
+
+    /// Pass a key/value (key=value) to all components of the application.
+    #[clap(short, long, parse(try_from_str = parse_kv))]
+    pub key_value: Vec<(String, String)>,
```

Comment:
nit seems this should be plural
```suggestion
    pub key_values: Vec<(String, String)>,
```